### PR TITLE
ICU-21900 ICU4C 71 change report

### PR DESCRIPTION
**Pull request by sven-oly**
##### Checklist

- [x] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-21900
- [x] Required: The PR title must be prefixed with a JIRA Issue number. 
- [x] Required: The PR description must include the link to the Jira Issue, for example by completing the URL in the first checklist item
- [x] Required: Each commit message must be prefixed with a JIRA Issue number. 
- [x] Issue accepted (done by Technical Committee after discussion)
- [x] Tests included, if applicable
- [x] API docs and/or User Guide docs changed or added, if applicable

### CODE REVIEW

**Comment by sven-oly** [comment]
Note that this does not refer to minor version numbers, just ICU 71.

**Review by richgillam — DISMISSED** [review]
LGTM.

File: APIChangeReport.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+
+  
+<!--
+ Copyright © 2019 and later: Unicode, Inc. and others.
+ License & terms of use: http://www.unicode.org/copyright.html
+-->
+
+# ICU4C API Comparison: ICU 70 with ICU 71 (m1)
```
Comment (markusicu) [inline APIChangeReport.md:8]: FYI it does say "(m1)" rather than just 71...

**Review by markusicu — CHANGES_REQUESTED** [review]
You are creating these as new files in the repo root, rather than modifying the files in icu4c/

**Comment by sven-oly** [comment]
Removed indicator of minor in report. PTAL.

**Comment by sven-oly** [comment]
Updated file that generates the change reports in order to remove minor and micro version info from API change report outputs.

File: APIChangeReport.md (hunk 1 above)
Comment (sven-oly) [reply APIChangeReport.md]: Thx. Fixed now.

**Review by markusicu — APPROVED** [review]
lgtm

**Pull request merged**